templates/core.j2

```diff
@@ -83,6 +83,9 @@ timer_resolution {{ main['timer_resolution'] }};
 {% if main['working_directory'] is defined %}
 working_directory {{ main['working_directory'] }};
 {% endif %}
+{% if main['quic'] is defined and main['quic']['bpf'] is defined and main['quic']['bpf'] is boolean %}
+quic_bpf {{ main['quic']['bpf'] | ternary('on', 'off') }};
+{% endif %}
```

Comment:
As convenient as it is, this really should be included in the http3/quic template. Directives in this template belong exclusively to the `ngx_core_module`.

Instead, I would tweak the `nginx.conf.j2` template to something along the lines of
```jinja2
{% if nginx_config_main_template['config']['main'] is defined %}
{% from 'core.j2' import main with context %}
{{ main(nginx_config_main_template['config']['main']) }}
{%- endif %}
{% if nginx_config_main_template['config']['main']['quic'] is defined %}
{% from 'http/modules.j2' import main with context %}
...
{%- endif %}
```
And we can then use scopes (!) to make sure that only the `quic_bpf` parameter/directive can be used here.

Reply:
well, IMO, bpf settings should be moved to core module in nginx as well. But I get the point. I'll move it out.